conf/test.config

```diff
@@ -20,5 +20,5 @@ params {
     max_time   = '6.h'
 
     // Input data
-    input = "https://github.com/phac-nml/staramrnf/raw/dev/assets/samplesheet.csv"
+    input = "https://github.com/phac-nml/staramrnf/raw/add-additional-nf-test/assets/samplesheet.csv"
```

Comment:
You can actually lave this as `dev`, since there's no asserts that check the output of the `test.config`. Once this PR is merged into `dev` it will just automatically run with the correct files.